compiler/src/iree/compiler/Codegen/Utils/VectorOpUtils.h

```diff
@@ -49,8 +52,14 @@ class VectorContractOpInfo {
   SmallVector<int64_t> outMDims;
   SmallVector<int64_t> outNDims;
 
+  SmallVector<unsigned> lhsUnitDims;
+  SmallVector<unsigned> rhsUnitDims;
+  SmallVector<unsigned> accUnitDims;
```

Comment:
Because upstream contraction dims that we use, also uses unsigned. Also, AffineMaps use unsigned. This is probably an anti-pattern, but it is to be consistent with other usages here.